### DISCUSSION TITLE
labels explained

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/Allocator-Governance#205] [Q&A] [answered]
Hello, governance team!

Is there a manual or readme giving a quick overview of what each label means and general process of transitioning from one label to another? Thanks.

**Comment by Kevin-FF-USA** [comment, marked as answer]
Hello @Fatman13,
<img width="624" alt="image" src="https://github.com/user-attachments/assets/d5d504e1-3def-4af8-b016-93636aee1fdf">

Great discussion question.  I added descriptions to each label to help make it easier to understand.  

**General Process**
Allocator Organization self submits a request DataCap Refresh
Label - `Refresh` 
Label - `Awaiting Governance/Watchdog`

This will trigger comments from either the community or a Watchdog account to ask questions of the application for additional DataCap.  Once the issue has received comments from Watchdog.
Label - `Awaiting Response from Allocator`.

Once the Allocator pathway has answered any open questions or clarifications the request will then receive a full audit from the Governance team.
Label - `Diligence Audit in Process`

After the Governance team completes a full audit of the distributions and leaves a comment,  the request then receives a check by the RKH.
Label - `Awaiting RKH`

Refresh will then close out.
Should there be a separate question or issue AFTER the Governance review the issue will receive a tracking label for - `Needs Governance`

**Comment by Kevin-FF-USA** [comment]
Added some new labels recently, so adding here for transparency and clarifications.

For Allocator APPLICATIONS
These new labels designate the self described function of the pathway to be onboarded.  Used to sort and track applications requests by RFA and Manual.
<img width="285" alt="image" src="https://github.com/user-attachments/assets/4087c58c-7ccb-4ff4-bffb-34ca55a40b7f" />

**Comment by Kevin-FF-USA** [comment]
For REFRESH requests
These designate the amount of DataCap that was awarded to the pathway in this recent request.
<img width="284" alt="image" src="https://github.com/user-attachments/assets/b3939fcb-2c9e-48d1-a030-5b8e605cfcd0" />
